cl/favorites/views.py

```diff
@@ -173,3 +176,18 @@ async def view_tags(request, username):
             "private": False,
         },
     )
+
+
+@cache_page(30)  # Cache for 30 seconds
+async def open_prayers(request: HttpRequest) -> HttpResponse:
+    """Show the user top open prayer requests."""
+
+    top_prayers = await get_top_prayers()
+    return TemplateResponse(
+        request,
+        "top_prayers.html",
+        {
+            "top_prayers": top_prayers,
+            "private": False,  # temporary to prevent Google indexing
```

Comment:
I read in a comment that this page should be hidden from search engines?  If that's correct, I think it should be set to `True`.

Reply:
My understanding is Mike wanted it to be private for the time being until the other parts of the feature are complete (allowing users to request documents to be added to the list). I'll modify the code accordingly.